core/store/migrate/migrations/0250_ccip_token_prices_fix.sql

```diff
@@ -0,0 +1,49 @@
+-- +goose Up
+
+-- We need to re-create tables from scratch because of the unique constraint on tokens and chains selectors
```

Comment:
Would we want to migrate existing data from the old tables or doesn't matter cuz we'll fetch and upsert new data?

Reply:
It doesn't matter, because it's gonna be almost immediately populated by nodes, similar to the deployment of that feature